main.py

```diff
@@ -1,7 +1,30 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"),
+              ("Will", "Smith"), ("Jaden", "Smith"),
```

Comment:
Consider splitting the tuple for 'Will Smith' and 'Jaden Smith' into separate lines for better readability and consistency with the other entries.